nepal_compliance/nepali_num2words.py

```
@@ -0,0 +1,107 @@
+	"""
+	from num2words import num2words
+
+	if not in_million:
+		# Check if currency is NPR (Nepalese Rupee)
+		try:
+			defaults = frappe.defaults.get_defaults()
+			currency = defaults.get("currency")
+			country = defaults.get("country")
+			if currency == "NPR" or country == "Nepal":
+				return _nepali_in_words(int(integer))
+		except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)